### PR TITLE
[rds-postgres] - The Execution Fails when Table name is in camel case

### PR DESCRIPTION
**Pull request by lucass4**
#### Description 
I am submitting this PR to address an issue related to having a table name in camel case. 

#### Scenario
The execution fails due to extra double quotes automatically add to tableNames and foreign keys.

**Command:**
```
klepto steal -c .klepto.toml --from="postgres://user:password@db-klepto-test-cluster.cluster-asxxxxxxxsdasdasd.us-east-1.xxxxx.amazonaws.com/dbName?sslmode=disable" --to="postgres://user:password@db-klepto-destination.cluster-asdasdasdasdasdasd.us-east-1.rds.amazonaws.com/dbName?sslmode=disable" **--to-rds** --concurrency=50
```

#### Error example:
```
error=failed to re-create ForeignKey "TableName".TableNames_userID_users_fk: pq: insert or update on table "TableName" violates foreign key constraint "TableNames_userID_users_fk" 
```

### CODE REVIEW

**Comment by davnnis2003** [comment]
@lucass4 @lucasmdrs hey guys - u do realize that this repo is public right? :D

File: pkg/dumper/postgres/dumper.go [hunk 1]
```diff
@@ -47,6 +47,15 @@ func (d *pgDumper) DumpStructure(sql string) error {
...
 }
 
+// quoteIfNotQuoted this function avoid quote strings that are already quoted.
+func quoteIfNotQuoted(s string) string {
+	if len(s) > 0 && s[0] == '"' {
+		return s
+	}
+
+	return strconv.Quote(s)
+}
```
Comment (lucasmdrs) [inline pkg/dumper/postgres/dumper.go:57]: to ensure the double quotes, maybe we could `strings.Trim`  and leverage the `%q` formatting instead where is needed

**Comment by lucass4** [comment]
> @lucass4 @lucasmdrs hey guys - u do realize that this repo is public right? :D

Sorry, I think I am missing the point. what do you mean? I do realize it is a public project and a public repo.

File: pkg/dumper/postgres/dumper.go (hunk 1 above)
Comment (lucass4) [reply pkg/dumper/postgres/dumper.go:57]: Thanks for the feedback! For sure, I can update the code.

Just to make sure, are you suggesting we don't use the function and do something like:

```
query := fmt.Sprintf("ALTER TABLE %q ADD CONSTRAINT %q %s", strings.Trim(fk.tableName), strings.Trim(fk.constraintName), fk.constraintDefinition)
```

Comment (lucasmdrs) [reply pkg/dumper/postgres/dumper.go:57]: yes, the `strings.Trim` take `"` as an argument too, but you got it :wink:

**Review by lucasmdrs — APPROVED** [review]

**Comment by lucasmdrs** [comment]
Thank you for the contribution! @lucass4

**Pull request merged**